tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -298,7 +298,7 @@ def augmented_dickey_fuller(x, param):
     series sample. This feature calculator returns the value of the respective test statistic.
 
     See the statsmodels implementation for references and more details.
-    
+
```

Comment:
I am sorry, this was not on purpose. I was not able to see the changes in `git diff`. Something went wrong there.